.eslintrc.js

```diff
@@ -0,0 +1,215 @@
+const rules = {
+  'curly': ['error'],
```

Comment:
Single quotes not required for property names that are valid identifiers:
```suggestion
  curly: ['error'],
```
(and many below).

Reply:
done

Reply:
Looks like the linter wants consistently quoted properties. In TypeScript, it's best practice to quote keys you don't know at compile time. For example:

```ts
interface Test {
  knownKey: string;
  [key: string]: number;
}
const test: Test = {
  knownKey: 'value',
  'unknownOne': 1,
  'unknownTwo': 2
};
```

With that principle in mind, maybe it's valid to keep the keys that are defined by eslint outside of this file quoted.